docs/xdr/usecases/use_your_own_cti.md

```diff
@@ -214,3 +214,137 @@ Several values can be specified separated by commas. i.e. `Phishing,APT-29`
 11.   Connect to `LiveAPI` using a websocket to be notified in real time of events happening in the community
 12.    Keep only messages that are about our task
+
+### Import indicators from dict
+!!! note
+    This is the prefered way for API integration.
+
+A dedicated endpoint allows to import indicators by sending them as the payload of a POST request. It supports batch creation of indicators, **up to 100 at once**.
+
+**1. Endpoint URL**
+
+`POST https://api.sekoia.io/v2/inthreat/ioc-collections/{collection_id}/indicators`
+
+**2. Payload specification**
+
+!!! note
+    The following section is in pseudo-code and cannot be used as-is, please see [usage example](#upload-ioc-dict-example) for a fully working example.
+```py
+{
+    "default_fields": {
+        description: str | None = None,
+        valid_from: DateTime = to_rfc3339(datetime.now(UTC)),
+        valid_until: DateTime | None = None,
+        kill_chain_phases: [KillChainPhaseSchema] | None = None,
+        x_ic_related_threat_refs: [str] | None = None
+    },
+    "indicators": [
+        {
+            value: str,
+            valid_from: DateTime | None = None
+        }
+    ]
+}
+```
+
+- All fiends are optional except for `indicators`, `default_fields`, and at least one instance of `indicators.value`. Here is a minimum working example:
+```py
+{
+    "default_fields": {},
+    "indicators": [{"value": "1.2.3.4"}]
+}
+```
+
+- `kill_chain_phases` must contain valid STIX kill chain phases
+- `x_ic_related_threat_refs` must contain valid STIX IDs
+
+<a name="upload-ioc-dict-example"></a>
+
+**3. Usage example**
+
+```py
+import json
+
+import requests
+from websockets.sync.client import connect
+
+
+# 1. Build the indicators' payload
+payload = { # (1)!
+    "indicators": [
+        {"value": "94.24.86.201"},
+        {"value": "152.10.13.131"},
+        {"value": "122.42.160.85"},
+        {"value": "35.118.7.95"},
+    ],
+    "default_fields": {
+        "valid_from": "2015-05-07T00:00:00Z",
+        "valid_until": "2045-05-07T00:00:00Z",
+        "kill_chain_phases": [
+            {"kill_chain_name": "lockheed-martin-cyber-kill-chain", "phase_name": "reconnaissance"}
+        ],
+        "x_ic_related_threat_refs": ["threat-actor--xxx"],
+    },
+}
+
+# 2. Use your API key for authentication
+api_key = "{Your API Key}"  # (2)!
+headers = {
+    "authorization": f"Bearer {api_key}"
+}
+
+# 3. Send the upload request
+collection_id = "ioc-collection--xxx"  # (3)!
+res = requests.post(
+    f"https://api.sekoia.io/v2/inthreat/ioc-collections/{collection_id}/indicators/",
+    headers=headers,
+    json=payload,
+)
+res.raise_for_status()
+nb_uploaded_IOCs = res.json()["total"]
+
+print(f"Imported {nb_uploaded_IOCs} successfully")
+```
+
+1. Replace with your own values/threat refs/etc
+2. Replace the content of this string with your API Key
+3. Replace the content of this string with the id of the collection the IOC must be added to
+
+**4. Response example**
```

Comment:
In the query you want to create 4 iocs and in the response you have only one. IMHO you should be consistent between the two, for example by having 2 iocs in both the request and the response.